packages/flutter/lib/src/widgets/animated_list.dart

```diff
@@ -16,10 +15,10 @@ import 'sliver.dart';
 import 'ticker_provider.dart';
 
 /// Signature for the builder callback used by [AnimatedList].
-typedef AnimatedListItemBuilder = Widget Function(BuildContext context, int index, Animation<double> animation);
+typedef AnimatedListItemBuilder = Widget Function(BuildContext? context, int index, Animation<double> animation);
```

Comment:
for all builders, BuildContext should not be nullable

Reply:
Done